### PR TITLE
Update seekTranscript to use long instead of Date for time.

### PR DESCRIPTION
**Pull request by amuamushu**
Currently, seekTranscript uses Date. However, a recent pull request updated the rest of the code to use long.

### CODE REVIEW

File: src/main/webapp/timestamps.js [hunk 1]
```diff
@@ -42,11 +41,8 @@ function timestampToSeconds(timestampMs) {
...
 /**
- * @return {number} The {@code date} in seconds.
+ * Converts `seconds` into milliseconds.
  */
-// TODO: Remove this method once the pull request updating
-// Date to long is approved.
-function getDateInSeconds(date) {
-  return date.getSeconds() + date.getMinutes() * MINUTES_TO_SECONDS +
-      date.getHours() * HOURS_TO_SECONDS;
+function secondsToMilliseconds(seconds) {
```
Comment (jaysaleh) [inline src/main/webapp/timestamps.js:46]: nit: maybe rename `seconds` we already know seconds is being passes in by the function name

File: src/main/webapp/view/transcript/transcript.js [hunk 2]
```diff
@@ -88,8 +88,8 @@ function appendTextToList(transcriptLine, ulElement) {
   liElement.appendChild(hrElement);
   ulElement.appendChild(liElement);
   // Save the dates for seeking later.
```
Comment (n-wach) [inline src/main/webapp/view/transcript/transcript.js:90]: update/remove this comment

Comment (amuamushu) [reply src/main/webapp/view/transcript/transcript.js]: Thanks for reminding me! Removed.

File: src/main/webapp/timestamps.js (hunk 1 above)
Comment (amuamushu) [reply src/main/webapp/timestamps.js:46]: I think I'm going to keep it as `seconds` if that's okay because I want the function to be general.

Comment (billie-cat) [inline src/main/webapp/timestamps.js:46]: wasn't this code added by Nathan? Should this PR be synced?

Comment (amuamushu) [reply src/main/webapp/timestamps.js:46]: No, this was a function I added. His function converts milliseconds to seconds (the other way around)

**Pull request merged**